dojo/tools/rusty_hog/parser.py

```diff
@@ -0,0 +1,127 @@
+                    description += "\n**Commit message:** {}".format(vulnerability.get('commit'))
+                if vulnerability.get('commitHash') is not None:
+                    description += "\n**Commit hash:** {}".format(vulnerability.get('commitHash'))
+                if vulnerability.get('parent_commit_hash') is not None:
+                    description += "\n**Parent commit hash:** {}".format(vulnerability.get('parent_commit_hash'))
+                if vulnerability.get('old_file_id') is not None and vulnerability.get('new_file_id') is not None:
+                    description += "\n**Old and new file IDs:** {} - {}".format(
+                                    vulnerability.get('old_file_id'),
+                                    vulnerability.get('new_file_id'))
+                if vulnerability.get('old_line_num') is not None and vulnerability.get('new_line_num') is not None:
+                    description += "\n**Old and new line numbers:** {} - {}".format(
+                                    vulnerability.get('old_line_num'),
+                                    vulnerability.get('new_line_num'))
+            elif scanner == "Gottingen Hog":
+                """Gottingen Hog"""
+                description = "**This string was found:** {}".format(vulnerability.get('stringsFound'))
+                if vulnerability.get('issue_id') is not None:
+                    description += "\n**JIRA Issue ID:** {}".format(vulnerability.get('issue_id'))
+                if vulnerability.get('location') is not None:
+                    description += "\n**JIRA location:** {}".format(vulnerability.get('location'))
+                if vulnerability.get('url') is not None:
+                    description += "\n**JIRA url:** {}".format(vulnerability.get('url'))
+            """General - for all Rusty Hogs"""
+            file_path = vulnerability.get('path')
+            if vulnerability.get('date') is not None:
+                description += "\n**Date:** {}".format(vulnerability.get('date'))
+            """Finding Title"""
+            if scanner == "Choctaw Hog":
+                title = "{} found in Git path {} ({})".format(
+                        vulnerability.get('reason'),
+                        vulnerability.get('path'),
+                        vulnerability.get('commitHash'))
+            elif scanner == "Gottingen Hog":
+                title = "{} found in Jira ID {} ({})".format(
+                        vulnerability.get('reason'),
+                        vulnerability.get('issue_id'),
+                        vulnerability.get('location'))
+            # create the finding object
+            finding = Finding(
+                title=title,
+                severity='High',
+                cwe=cwe,
+                description=description,
+                mitigation="Please ensure no secret material nor confidential information is kept in clear within git repositories.",
```

Comment:
Just saw this while testing the PR. The mitigation text shall be different for each parser. For Gottingen Hog it shall reference to Jira and not git.

Apart from this the PR is fine for me.